test/e2e/common/podtemplates.go

```diff
@@ -33,9 +33,9 @@ var _ = ginkgo.Describe("[sig-architecture] PodTemplates", func() {
 	f := framework.NewDefaultFramework("podtemplate")
 	/*
 	   Release : v1.18
-	   Testname: ConfigMap lifecycle
-	   Description: Attempt to create a ConfigMap. Patch the created ConfigMap. Fetching the ConfigMap MUST reflect changes.
-	          By fetching all the ConfigMaps via a Label selector it MUST find the ConfigMap by it's static label and updated value. The ConfigMap must be deleted by Collection.
+	   Testname: PodTemplate lifecycle
+	   Description: Attempt to create a PodTemplate. Patch the created PodTemplate. Fetching the PodTemplate MUST reflect changes.
+	          By fetching all the PodTemplates via a Label selector it MUST find the PodTemplate by it's static label and updated value. The PodTemplate must be deleted by Collection.
```

Comment:
Shouldn't it be using DeleteCollection not Delete, based on this comment? Should we add one more template, delete one by `Delete` and the rest by `DeleteCollection`? Would that get us another endpoint?

Reply:
You're right, according to my original ticket I was meant include `DeleteCollection` instead of `Delete`. Should this be fixed in a new PR and the promotion be delayed?